### PR TITLE
NHAIS Azure TF

### PR DESCRIPTION
**Pull request by LukaszKwasniewski**

### CODE REVIEW

**Review by michalschott — DISMISSED** [review]

File: terraform/azure/components/account/instance_jumpbox.tf [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  admin_ssh_key {
+    username = var.jumpbox_user
+    public_key = file("files/azure_account_jumpbox.pub")
+  }
+
+  os_disk {
+    name                 = "${local.resource_prefix}-jumpbox_disk"
+    caching              = "ReadWrite"
+    storage_account_type = "Premium_LRS"
```
Comment (mattd-kainos) [inline terraform/azure/components/account/instance_jumpbox.tf]: Do we need Premium LRS or is there a less expensive option e.g. Standard LRS?

File: terraform/azure/components/base/instance_testbox.tf [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+  admin_ssh_key {
+    username = var.jumpbox_user
+    public_key = file("../account/files/azure_account_jumpbox.pub")
+  }
+
+  os_disk {
+    name                 = "${local.resource_prefix}-testbox_disk"
+    caching              = "ReadWrite"
+    storage_account_type = "Premium_LRS"
```
Comment (mattd-kainos) [inline terraform/azure/components/base/instance_testbox.tf]: as for jumpbox about standard vs premium

File: terraform/azure/components/base/servicebus.tf [hunk 3]
```diff
@@ -0,0 +1,11 @@
+resource "azurerm_servicebus_namespace" "base_servicebus_namespace" {
+  name                = "${local.resource_prefix}-service-bus"
+  location            = var.location
+  resource_group_name = var.account_resource_group
+  capacity            = 1
+  sku                 = "Premium"
```
Comment (mattd-kainos) [inline terraform/azure/components/base/servicebus.tf]: Do we need to use a Premium ASB instance?

File: terraform/azure/components/nhais/kube-nhais_deployment.tf [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+          }
+
+          env {
+              name = "NHAIS_MESH_SHARED_KEY"
+              value = var.nhais_mesh_shared_key
+          }
+
+          env {
+              name = "NHAIS_MESH_HOST"
+              value = var.fake_mesh_in_use ? "https://${local.resource_prefix}-fake-mesh:${var.fake_mesh_application_port}/messageexchange/" : var.nhais_mesh_host
```
Comment (mattd-kainos) [inline terraform/azure/components/nhais/kube-nhais_deployment.tf:122]: 👍 I was thinking how to do GP2GP mock component - seperate or flagged in/out in the gp2gp component.

**Review by mattd-kainos — COMMENTED** [review]
Consider what storage and ASB tiers should be used. Or should we put these in vars?

**Review by mattd-kainos — APPROVED** [review]

**Pull request merged**